pkg/controller/routes.go

```diff
@@ -110,7 +110,8 @@ func routes(router *router.Router, cfg *rest.Config, registryTransport http.Roun
 	router.Type(&corev1.Pod{}).Selector(managedSelector).HandlerFunc(jobs.JobPodOrphanCleanup)
 	router.Type(&corev1.Pod{}).Selector(managedSelector).HandlerFunc(jobsHandler.SaveJobOutput)
 	router.Type(&netv1.Ingress{}).Selector(managedSelector).Namespace(system.ImagesNamespace).HandlerFunc(gc.GCOrphans)
-	router.Type(&netv1.Ingress{}).Selector(managedSelector).Middleware(ingress.RequireLBs).Handler(ingress.NewDNSHandler())
+	router.Type(&netv1.Ingress{}).Selector(managedSelector).Namespace(system.Namespace).Middleware(ingress.RequireLBs).Handler(ingress.NewDNSHandler())
```

Comment:
If we only care about acting something with a specific name, you can use `.Name("acorn-dns-name")` in addition to the `Namespace` method. This will means you can get rid of the name check in the method.

Reply:
Updated!

Reply:
oh good. ...I thought there should be a Name() but didnt see an example in here.